src/OrchardCore.Modules/OrchardCore.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -99,5 +109,13 @@ private IEnumerable<ContentTypeDefinition> GetContainedContentTypes(BuildPartDis
             var contentTypes = settings.ContainedContentTypes ?? Enumerable.Empty<string>();
             return contentTypes.Select(contentType => _contentDefinitionManager.GetTypeDefinition(contentType));
         }
+
+        private async Task<bool> IsContentCreatableAsync(ContentTypeDefinition contentTypeDefinition)
+        {
+            return await _authorizationService.AuthorizeAsync(
+                _httpContextAccessor.HttpContext.User,
+                CommonPermissions.EditContent,
+                contentTypeDefinition);
```

Comment:
Instead of trying to authorize against the first content type definition (does that actually work?) is should authorize against a new content item of each, how `AdminController` in the Contents module does.